### PR TITLE
Option for centering content

### PR DESCRIPTION
**Pull request by KashishMistry**
### Related Item(s)
[#403](https://github.com/ramp4-pcar4/story-ramp/issues/403) in `story-ramp` repo

### Changes
- This issue is logged in the `story-ramp` repo, however the changes needed to be implemented in `storylines-editor` to allow users to manipulate the `customStyles` property for each panel
- `centerPanel` allows the user to center content in each panel 
- `centerSlide` allows the user to center content in the overall slide (if the slide has two panels then the left panel is right-aligned and the right panel is left-aligned)

### Notes
Without centering:
alignment normal

When `centerPanel` is selected:
alignment panel

When `centerSlide` is selected:
alignment slide

When there is only one panel and `centerSlide` or `centerPanel` is selected:
alignment one

### Testing
Steps:
1. Create a new slide
2. Select `Center panel content` and preview to check the alignment of the content on the panel (should be centered relative to the panel).
3. Select `Center slide content` and preview to check the alignment of the content on the slide (left slide should be right-aligned and right slide should be left-aligned).
4. Experiment with dynamic panels in particular to ensure the panels in `children` are being aligned correctly.

- - -
This change is [Reviewable](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285)

### CODE REVIEW

**Review by yileifeng — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285)* status: 0 of 9 files reviewed, 2 unresolved discussions (waiting on @KashishMistry)

___
*[`src/components/editor/slide-editor.vue` line 40 at r1](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285#-NwFHSuW2AUIuUa4DhrL:-NwFHSuW2AUIuUa4DhrM:bazym3w) ([raw file](https://github.com/ramp4-pcar4/storylines-editor/blob/2b74c74e1bb659c3543f886a63ca76c43640cf74/src/components/editor/slide-editor.vue#L40)):*
> ```Vue
>                         />
>                         <span class="mx-2 font-bold">{{ $t('editor.slides.centerSlide') }}</span>
>                         <input
> ```



The toggle checkboxes are not aligned with the text
___
*[`src/lang/lang.csv` line 115 at r1](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285#-NwFFrvvC5zdaGAJE7oG:-NwFFrvvC5zdaGAJE7oH:bdd3yat) ([raw file](https://github.com/ramp4-pcar4/storylines-editor/blob/2b74c74e1bb659c3543f886a63ca76c43640cf74/src/lang/lang.csv#L115)):*
> ```CSV
> editor.slides.makeFull,Make the right panel the full slide,1,Mettre la diapositive complète dans le panneau de droite,1
> editor.slides.centerPanel,Align panel content towards the middle,1,Aligner le contenu du panneau vers le milieu,0
> editor.slides.centerSlide,Align slide content towards the middle,1,Aligner le contenu de la diapositive vers le milieu,0
> ```

Suggestion: how about rewording this to "Center panel content" and "Center slide content"? Would preserve some valuable space as well and could be more intuitive wording.

**Review by KashishMistry — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285)* status: 0 of 9 files reviewed, 2 unresolved discussions (waiting on @yileifeng)

___
*[`src/components/editor/slide-editor.vue` line 40 at r1](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285#-NwFHSuW2AUIuUa4DhrL:-NwFM4GE5NQnxerl8ZCg:b-ugpthe) ([raw file](https://github.com/ramp4-pcar4/storylines-editor/blob/2b74c74e1bb659c3543f886a63ca76c43640cf74/src/components/editor/slide-editor.vue#L40)):*
<details><summary><i>Previously, yileifeng (Yi Lei Feng) wrote…</i></summary><blockquote>



The toggle checkboxes are not aligned with the text
</blockquote></details>

Donethanks.
___
*[`src/lang/lang.csv` line 115 at r1](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285#-NwFFrvvC5zdaGAJE7oG:-NwFM5Qi7mYVdxkCaYY7:b-ugpthe) ([raw file](https://github.com/ramp4-pcar4/storylines-editor/blob/2b74c74e1bb659c3543f886a63ca76c43640cf74/src/lang/lang.csv#L115)):*
<details><summary><i>Previously, yileifeng (Yi Lei Feng) wrote…</i></summary><blockquote>

Suggestion: how about rewording this to "Center panel content" and "Center slide content"? Would preserve some valuable space as well and could be more intuitive wording.
</blockquote></details>

Donethanks.

**Review by yileifeng — APPROVED** [review]
> Reviewed 7 of 9 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @KashishMistry)

**Review by dnlnashed — APPROVED** [review]
> Reviewed 7 of 9 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @KashishMistry)

**Review by RyanCoulsonCA — APPROVED** [review]
> Reviewed 7 of 9 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/285)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @KashishMistry)
